inputfiles/sherpa/LHC-LO-ZplusJet/Run.dat

```diff
@@ -18,11 +18,12 @@
 (processes){
   Process 93 93 -> 13 -13 93;
   Order (*,2);
+  CKKW sqr(QCUT/E_CMS);
+  Cut_Core 1;
```

Comment:
I thought this is only for NLO matching?

Reply:
Turns out, after discussion with Sherpa authors, this is the undocumented but preferred way to set the scales.